### ISSUE TITLE
A11Y Message in wifi password box should be focusable

### ISSUE THREAD

**Issue by paul-geoghegan** [canonical/ubuntu-desktop-provision#1419]
In my opinion the message in the wifi password dialog should be focusable via keyboard so that Orca users can read the message. I'll happily create a PR for this myself.

**Comment by paul-geoghegan** [comment]
I'm actually going to close this as it turns out that that dialog seems to be a GTK native one so you can actually use flat review to read it.
